osg-xrootd/osg/10-common-site-local.cfg

```diff
@@ -20,4 +20,10 @@
 # match the registered resource name in topology.opensciencegrid.org.
 # See https://github.com/opensciencegrid/topology/#topology
 #
-# set resourcename = OSG_RESOURCE_NAME
+# set resourcename = OSG_RESOURCE_NAM
+
+# We suggest setting up an instance of the xrootd-monitoring-shoveler
+# in order to prevent UDP packet lose due to fragmentation.
+# If you have done that, specify its host and port in ShovelerHostPort
+# https://github.com/opensciencegrid/xrootd-monitoring-shoveler
```

Comment:
```suggestion
# set resourcename = OSG_RESOURCE_NAME

# We suggest setting up an instance of the xrootd-monitoring-shoveler
# in order to prevent UDP packet loss due to fragmentation.
# If you have done that, specify its host and port in ShovelerHostPort
# https://github.com/opensciencegrid/xrootd-monitoring-shoveler
```